src/main/java/step2/CalculateOperator.java

```diff
@@ -0,0 +1,35 @@
+package step2;
+
+import java.util.Arrays;
+import java.util.function.BiFunction;
+
+public enum CalculateOperator {
+  PLUS("+", (x, y) -> x + y),
+  MINUS("-", (x, y) -> x - y),
+  MULTIPLY("*", (x, y) -> x * y),
+  DIVIDE("/", (x, y) -> {
+    if (y == 0) {
+      throw new IllegalArgumentException();
+    }
+    return x / y;
+  });
+
+  private String operator;
```

Comment:
```suggestion
  private final String operator;
```
enum이니 final 붙여도 문제 없겠네요